test/versioned/winston-esm/package.json

```diff
@@ -9,7 +9,7 @@
         "node": ">=16"
       },
       "dependencies": {
-        "winston": ">=3",
+        "winston": ">=3.4.0",
```

Comment:
shouldn't you just wrap the alias import test in a semver check? the named exports still work pre 3.4